data/net/minecraft/util/Mth.mapping

```diff
@@ -97,6 +129,18 @@ CLASS net/minecraft/util/Mth
 			COMMENT End value for the lerp.
 		ARG 2 delta
 			COMMENT A value between 0 and 1 that indicates the percentage of the lerp. (0 will give the start value and 1 will give the end value) If the value is not between 0 and 1, it is clamped.
+	METHOD clampedMap (DDDDD)D
+		ARG 0 input
+		ARG 2 inputMin
+		ARG 4 inputMax
+		ARG 6 ouputMin
+		ARG 8 outputMax
```

Comment:
This doesn't make as much sense to me. I understand what you're going for as the first three take in the from and to values to get the delta and then maps it to the output. I don't know what would be clearer though without a javadoc.